BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs

```diff
@@ -79,6 +79,22 @@ public Guid ConnectToDepthWebSocket(string symbol, BinanceWebSocketMessageHandle
             var endpoint = new Uri($"{BaseWebsocketUri}/{symbol.ToLower()}@depth");
             return CreateBinanceWebSocket(endpoint, messageEventHandler);
         }
+
+        /// <summary>
+        /// Connect to thePartial Book Depth Streams
+        /// </summary>
+        /// <param name="symbol"></param>
+        /// <param name="messageEventHandler"></param>
+        /// <returns></returns>
+        public Guid ConnectToPartialDepthWebSocket(string symbol,int levels, BinanceWebSocketMessageHandler<BinancePartialData> messageEventHandler)
```

Comment:
You can use `ConnectToDepthWebSocketCombinedPartial` method to do same, just use one symbol.